app/personal_values/static/value_descriptions.json

```diff
@@ -27,30 +27,30 @@
     "id": "self_direction",
     "name": "self direction",
     "description": "At its essence, Self-Direction is a need for control, mastery, and independence - so it’s no surprise you might relish in choosing your own goals - and honing till they are just right. Likely curious, intelligent and a lover of freedom - you prefer to live by your own rules.",
-    "shortDescription": "You value freedom of thought and action, prefering to come to conclusions or decisions yourself. You also likely draw great satisfaction when creating or exploring the world."
+    "shortDescription": "You value freedom of thought and action, preferring to come to conclusions or decisions yourself. You also likely draw great satisfaction when creating or exploring the world."
   },
   "stimulation": {
     "id": "stimulation",
     "name": "stimulation",
-    "description": "The world is a big place and you see the benefits of craving new experiences. On top of a need for variety and stimulation, you may also be a bit daring compared to most. You are likely a go-getter change-maker at heart.",
+    "description": "The world is a big place and you see the benefits of craving new experiences. On top of a need for variety and stimulation, you may also be a bit daring compared to most. You are likely a go-getter and change-maker at heart.",
```

Comment:
This is an okay change to make, or could just add a comma instead (either would be okay to do).